swarm/storage/mru/timestampprovider.go

```diff
@@ -19,24 +19,19 @@ package mru
 import (
 	"encoding/binary"
 	"time"
-
-	"github.com/ethereum/go-ethereum/common"
 )
 
-// Encodes a point in time as a Unix epoch and provides
-// space for proof the timestamp was not created prior to its time
+// Encodes a point in time as a Unix epoch
 type Timestamp struct {
-	Time  uint64      // Unix epoch timestamp, in seconds
-	Proof common.Hash // space to hold proof of the timestamp, e.g., a block hash
+	Time uint64 // Unix epoch timestamp, in seconds
```

Comment:
don't need anymore, right?

Reply:
I'd rather keep the structure for the reasons stated above, makes it very easy to extend or modify in the future + a place for the serializers to be.